index.html

```diff
@@ -6,17 +6,21 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Christabelle's Portfolio</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.8.1/font/bootstrap-icons.css">
      <link rel="stylesheet" href="style.css"> 
     <link rel="stylesheet" href="desk.css"> 
 </head>
 
 <body>
-    <main class="maincontainer">
+    <main class="maincontainer" id="main1">
         <header>
             <nav class="navbar">
                 <a href="#" class="logo">Crysty</a>
-                <a href="#" class="menu"><img src="images/Union.png" alt="dropdown menu"> </a>
-                <ul class="nav_items">
+                <div class="menu3">
```

Comment:
- [ ] Kindly adjust your blur effect and the color to match the design given, below is a comparison of your design and the Figma design.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/61408860/167661954-255187a9-adb8-4572-bc1c-cba0a626eb78.png) | ![image](https://user-images.githubusercontent.com/61408860/167662041-7ebcfcc6-32dc-4e71-9b7d-2e0e27883d54.png)